travis.cfg

```diff
@@ -1,12 +0,0 @@
-[buildout]
-extends = buildout.cfg
-parts += createcoverage
-
-[code-analysis]
-recipe = plone.recipe.codeanalysis
-pre-commit-hook = False
-return-status-codes = True
```

Comment:
@hvelarde if we remove travis.cfg p.r.codeanalysis won't be able to return proper status codes on travis.